CMakeLists.txt

```diff
@@ -207,8 +207,9 @@ if(LIBREMIDI_EXAMPLES)
   endif()
 endif()
 
-find_package(Catch2)
+find_package(Catch2 QUIET)
 if(TARGET Catch2::Catch2 AND Catch2_FOUND)
+  message(" -- libremidi : compiling tests")
```

Comment:
with CMake we can use message(STATUS "the message") instead of adding " -- ", it does the same thing but also allows to filter messages depending on their log level :)